### DISCUSSION TITLE
问题反馈

### DISCUSSION THREAD

**Discussion by Z4Knight** [ZuodaoTech/everyone-can-use-english#1151] [General]
hello，反馈一个小 bug，当我选中一些短语进行反复练习时，会出现播放后不能停止的情况。我本来是想要循环练习这个短语的，结果它会继续往下读，除非我去手动重新播放一下，它才会再从我选中的短语开始读。

另外，我测试下来，0.5.0 这个版本以后，才有的这个小 bug，最新版本 0.6.1 还是存在。

PS: 我的电脑型号是 Mac book pro 21 款，操作系统是 14.4.1。

![1729993052934](https://github.com/user-attachments/assets/ab2b55a8-a0db-44ad-9547-bf2473b8c838)

**Comment by an-lee** [comment]
是部分录音会出现这种情况是吗？

**Reply by Z4Knight** [reply]
不是的，所有录音都会出现。只是出现的频率随机，有的时候是刚播放就出现，有的是循环播了好几次才会出现。对了，我的录音文件是通过和智能助手对话以后生成的，也就是我把中文发给了 GPT 以后，让它帮我生成英文和语音后，我直接跟读的。
